### ISSUE TITLE
"Feature Request" Gears

### ISSUE THREAD

**Issue by lilmoefow** [dmaster8/DMTweaks#1]
Do you think it would be possible to get a gear for anything that registers as an ingot in the ore dictionary? or just some more gear types? maybe Thaumcraft and Extra Utilites support? Some examples, Thaumium, Void, Unstable, Bedrockium.

**Comment by dmaster8** [comment]
Sounds cool

**Comment by lilmoefow** [comment]
I actually just finished making texture for some if you want to check them out.. I plan on using them with Quadrum in my "private" modpack that im making for me and my friends. I only made textures for Unstable, Bedrockium, Thaumium, Void(not to happy with this ones outcome), AncientWarefares Steel, and Elementium(because you were missing that one :P)   I basically just took the gear texture from thermalfoundation and edited it to match the ingot textures.

**Comment by dmaster8** [comment]
Yeah, I'm working on an update to add these gears, so the textures would be cool :)

**Comment by lilmoefow** [comment]
-Bedrockium
-Elementium
-Steel(Ancient Warefare Texture)
-Thaumium
-Unstable
-Void
http://imgur.com/a/kOvRM

ill dropbox links to the actual files if you like the looks of those

**Comment by ghost** [comment]
Was this ever completed?

I would like to request support for:
Mekanism
Metallurgy
